models/label.rb

```diff
@@ -0,0 +1,18 @@
+class Label
+  attr_reader :id
+  attr_accessor :title, :color, :items
+
+  def initialize(title:, color:, id: nil)
+    @title = title
+    @color = color
+    @id = id
+    @items = []
+  end
+
+  private
+

```

Comment:
- The `add_item` method is a public method according to the **[UML Diagram](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/images/catalog_of_my_things.png)**
- Please just remove this line that making the method `private` and you are good to go. 🥇